build.gradle

```diff
@@ -7,10 +7,10 @@ buildscript {
     apply from: 'build-tools/repositories.gradle'
 
     ext {
-        opensearch_version = System.getProperty("opensearch.version", "2.2.0-SNAPSHOT")
+        opensearch_version = System.getProperty("opensearch.version", "2.4.0-SNAPSHOT")
```

Comment:
Let's also note in the PR description that we're bumping `main` to `2.4.0` here if it's necessary for it to be in this PR. Ideally, we should separate it into another PR so it's easier to note when it happened in the commit history at a glance.

Reply:
It's necessary in this PR. Have updated the description as suggested. thanks for pointing it out.

Reply:
Cool, spoke with @eirsep and we'll merge in https://github.com/opensearch-project/alerting/pull/596 and then this PR.